### ISSUE TITLE
Add attribute with list of cards/components with updates available

### ISSUE THREAD

**Issue by pnbruckner** [custom-components/custom_updater#28]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
Add an attribute that contains a list (or tuple) of all the cards/components that have updates available. This would be a nice summary, in addition to the state that shows the count. For people not using the tracker card this could be displayed in a template sensor. And for the tracker card, maybe this would be nice to display as well.

**Describe alternatives you've considered**
A template sensor _could_ display this information already, but it would be nice if custom_updater did this itself. E.g.:
```
sensor:
  - platform: template
    sensors:
      custom_updates:
        entity_id: sensor.custom_component_tracker
        value_template: >
          {% for attr in states.sensor.custom_component_tracker.attributes.items()
              if attr[1].has_update -%}
            {% if not loop.first %},{% endif %}{{ attr[0] }}
          {%- endfor %}
```

**Additional context**
None.

**Comment by ludeeus** [comment]
I agree!

**Comment by ludeeus** [comment]
![image](https://user-images.githubusercontent.com/15093472/46154622-da0c4700-c275-11e8-9c38-39315ce64ae3.png)
Adding it this way for next version, I hope that will suite your needs :)

**Comment by ludeeus** [comment]
Added in 2.7.2 <https://github.com/custom-components/custom_updater/releases/tag/2.7.2>

**Comment by pnbruckner** [comment]
Looks good. Thanks!

**Comment by ludeeus** [comment]
@pnbruckner Thank you for this idea!
I'm in the progress of rewriting the component, and I can remove a lot of code because of this 👍

**Comment by pnbruckner** [comment]
FWIW, I just noticed that this attribute was correctly indicating one of my custom components had an update. But after updating, with the local and remote versions being the same, the component is still listed in the has_update attribute. Should I open a new issue?

**Comment by pnbruckner** [comment]
Since I'm working on a PR to fix issue #31, I'll just fix this, too.

**Comment by ludeeus** [comment]
@pnbruckner That is a caching issue that I have resolved in 3.0.0
